engine/troubleshoot.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Troubleshoot Docker Engine
+description:
+  Diagnose and resolve error messages related to the Docker Engine installation
+keywords: Docker Engine, troubleshooting, error, Linux
+---
+
+This page contains instructions for troubleshooting and diagnosing the Docker
+Engine installation.
+
+## Kernel compatibility
+
+Docker can't run correctly if your kernel is older than version 3.10, or if it's
+missing kernel modules. To check kernel compatibility, you can download and run
+the
+[`check-config.sh`](https://raw.githubusercontent.com/docker/docker/master/contrib/check-config.sh)
+script.
+
+```console
+$ curl https://raw.githubusercontent.com/docker/docker/master/contrib/check-config.sh > check-config.sh
+
+$ bash ./check-config.sh
+```
+
+The script only works on Linux.
+
+## `Cannot connect to the Docker daemon`
```

Comment:
This formatting took me out of the documentation for a brief second and in some funny way looks like it is part of the section above. I understand why it is like this (and I know it was like this in the old version), but perhaps it should be a normal h2 'Unable to connect .....' and then leads into the error below. 
A v. minor point though.